site/hxapi/transfers/bkg/index.md

```diff
@@ -0,0 +1,116 @@
+---
+
+---
+
+# Transfers Booking
+
+## Booking Request
+
+### Method
+
+POST
+
+### Endpoint
+
+The endpoint to use is:
+```
+https://api.holidayextras.co.uk/v1/transfers/[CODE]
```

Comment:
Does this `CODE` have a limit (e.g only valid for 10mins after the search)?

Reply:
It could vary per supplier, could be a hard one to try and explain at the moment. Will be interesting to see if we get any questions, as we have asked our suppliers this question before and they say it can vary.